File: app/src/main/java/com/bian/viewapplication/MainActivity.java
package com.bian.viewapplication;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.SurfaceView;
import android.widget.Button;

import com.bian.viewapplication.activity.ListViewActivity;
import com.bian.viewapplication.activity.RecyclerViewActivity;

public class MainActivity extends AppCompatActivity {
    private SurfaceView mSurfaceView;
    private Button listButton;
    private Button recyclerButton;
    private Button vgButton;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        initView();
    }

    private void initView() {
        listButton = findViewById(R.id.listview_bt);
        listButton.setOnClickListener(v -> startActivity(new Intent(this, ListViewActivity.class)));
        recyclerButton = findViewById(R.id.recycler_bt);
        recyclerButton.setOnClickListener(v -> startActivity(new Intent(this, RecyclerViewActivity.class)));
        vgButton = findViewById(R.id.vg_button);
    }
}

File: app/src/main/java/com/bian/viewapplication/MainActivity.kt
package com.bian.viewapplication

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.bian.viewapplication.activity.*
import com.bian.viewapplication.bean.BankInfo
import com.bian.viewapplication.util.CrashHandler
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        initView()
        testKotlin()
    }

    private fun testKotlin() {
        val bankInfo = BankInfo("反对共产党银行", R.drawable.logo_abc)
        val copy = bankInfo.copy(bankName = "dada")
    }

    private fun initView() {
        CrashHandler.getInstance().init(this)
        listview_bt.setOnClickListener {
            startActivity(Intent(applicationContext, NewListActivity::class.java))
        }
        vg_button.setOnClickListener { startActivity(Intent(this, ViewGroupActivity::class.java)) }
        refresh_button.setOnClickListener { startActivity(Intent(this, RefreshListActivity::class.java)) }
        behavior_bt.setOnClickListener { startActivity(Intent(this, BehaviorActivity::class.java)) }
        bianView.setOnClickListener { startActivity(Intent(this, ToolbarActivity::class.java)) }
        bt_eyepetizer.setOnClickListener{startActivity(Intent(this,EyepetizerActivity::class.java))}
        bt_multi_type.setOnClickListener { startActivity(Intent(this,MultiItemActivity::class.java)) }
    }
}
